src/bitwarden_exporter/__main__.py

```
@@ -27,6 +27,65 @@
+    Args:
+        secret_path: The secret identifier with optional prefix
+        all_items_list: List of Bitwarden items for JMESPath evaluation
+
+    Returns:
+        The resolved secret string
+    """
+
+    if secret_path.startswith("env:"):
+        env_password = os.getenv(secret_path[4:])
+        if env_password is None or len(env_password) == 0:
+            raise ValueError(f"Environment variable not found: {secret_path}")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)